android/build.gradle

```diff
@@ -123,80 +118,84 @@ android {
-            proguardFiles getDefaultProguardFile('proguard-android.txt'),
-                    'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
             multiDexKeepFile file('multidex-config.txt')
             //multiDexKeepProguard file('proguard-rules.pro')
         }
     }
 
-
     flavorDimensions "classicApp"
-
     productFlavors {
         basic {
-            applicationId 'com.frostwire.android'
             versionCode 9060000 + project.ext.versionCode
         }
         plus1 {
-            applicationId 'com.frostwire.android'
             versionCode 9070000 + project.ext.versionCode
         }
     }
 
     applicationVariants.all { variant ->
         changeApkOutput(variant)
     }
+
+    lintOptions {
+        abortOnError false
+    }
+
+    compileOptions {
+        encoding = 'UTF-8'
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    dexOptions {
+        javaMaxHeapSize "2g"
+        jumboMode = true
+    }
+
+    signingConfigs {
+        release {
+            storeFile file("../../frostwire-tools/certs/android.keystore/frostwire.keystore")
+            storePassword ""
+            keyAlias ""
+            keyPassword ""
+        }
+    }
 }
 
 dependencies {
+    implementation 'com.android.support:animated-vector-drawable:28.0.0'
+    implementation 'com.android.support:customtabs:28.0.0'
+    implementation 'com.android.support:design:28.0.0'
+    implementation 'com.android.support:exifinterface:28.0.0'
+    implementation 'com.android.support:preference-v7:28.0.0'
+
+    implementation 'com.android.billingclient:billing:1.2'
     implementation 'com.android.support:multidex:1.0.3'
+    implementation 'com.google.android.gms:play-services-ads:17.1.2'
+    implementation 'com.google.android:flexbox:1.0.0' //1.1.0
 
-    implementation 'com.android.support:design:27.1.1'
-    implementation 'com.android.support:animated-vector-drawable:27.1.1'
-    implementation 'com.android.support:customtabs:27.1.1'
-    implementation 'com.android.support:exifinterface:27.1.1'
-    implementation 'com.android.support:preference-v7:27.1.1'
-    implementation 'com.android.billingclient:billing:1.1'
-    implementation 'com.google.android.gms:play-services-ads:17.1.1'
-    implementation('com.mopub:mopub-sdk-banner:5.4.1@aar') { transitive = true }
-    implementation('com.mopub:mopub-sdk-interstitial:5.4.1@aar') { transitive = true }
-    implementation 'com.applovin:applovin-sdk:9.1.3'
-    implementation 'com.squareup.picasso:picasso:3.0.0-SNAPSHOT'
     implementation 'com.frostwire:jlibtorrent-android-arm:1.2.0.18'
     implementation 'com.frostwire:jlibtorrent-android-x86:1.2.0.18'
-    implementation 'com.google.android:flexbox:1.0.0'
+
+    implementation 'com.squareup.picasso:picasso:3.0.0-SNAPSHOT'
+    //implementation 'com.squareup.picasso:picasso3:3.0.0-SNAPSHOT'
```

Comment:
not sure what you did here

Reply:
Just a little note that picasso is going to move to picasso3 (and also where the newer snapshot releases are, so... just get ready to migrate that I guess)